zstacklib/zstacklib/utils/lv_metadata_handler.py

```
@@ -0,0 +1,219 @@
+        vm_metadata.initialize_metadata_lv(
+            metadata_path, vm_metadata.INITIAL_LV_SIZE)
+        logger.info("created and initialized metadata LV %s", metadata_path)
+
+    def _lv_list_func(self, vg):
+        """Return list of ``(lv_name, lv_path, lv_size)`` tuples for *vg*.
+
+        Decorated with ``@bash.in_bash`` at call-site via the injected bash
+        module; here we define the plain function and the constructor caller
+        wraps it.
+        """
+        r, o = self._bash.bash_ro(
```

Ruff (0.15.6)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)